sdk/ai/azure-ai-generative/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## 1.0.0b1 (unreleased)
```

Comment:
Follow the structure shown here with "Release History" at the top or the CI will scream

https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/template/azure-template/CHANGELOG.md?plain=1

Reply:
Also, I cannot comment on the file, but recommend adding at least the template for the README.md to similarly appease CI: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/template/azure-template/README.md

Reply:
Added